### ISSUE TITLE
SpinButton doesn't increment or decrement on Voiceover VO controls

### ISSUE THREAD

**Issue by Ashwin-Nikam** [microsoft/fluentui#13606]
### Environment Information

- **Package version(s)**: 7.116.1
- **Browser and OS versions**: Safari, Mac OS Catalina 10.15.5

### Describe the issue:
The standard VoiceOver modifier is the control + option keys pressed at the same time. For using VoiceOver with steppers in order to increment or decrement the stepper VoiceOver instructs us to use control+option+up/down arrow key. On using this with the SpinButton, the VoiceOver annouces that the value is incrementing/decrementing but the actual value doesn't change. 

#### Actual behavior:

When VoiceOver is enabled and we're currently inside the SpinButton, pressing control + option + up/down arrow key announces that stepper is incrementing/decrementing but actual value doesn't change.

#### Expected behavior:

When VoiceOver is enabled and we're inside the SpinButton, pressing control + option + up/down arrow key should announce that stepper is incrementing/decrementing and the actual value should change as well.

### Documentation describing expected behavior

https://www.apple.com/voiceover/info/guide/_1121.html

When VoiceOver is on, you can use VoiceOver commands to navigate and interact with items on the screen. You enter VoiceOver commands by holding down the Control and Option keys together, along with one or more other keys. The Control and Option keys are called the “VoiceOver keys,” or “VO keys” for short. They are shown in commands as VO, as in VO-F1. You can assign VoiceOver commands to numeric keypad keys, keyboard keys, braille display input keys, and trackpad gestures, so you can use the commands with fewer keystrokes.

**Comment by Ashwin-Nikam** [comment]
Hey @paulgildea @khmakoto do we have any updates on this? 😀 I was hoping this could be prioritized as we need accessibility sign off for a feature in OWA.

**Comment by khmakoto** [comment]
@paulgildea I don't have a Mac with which to test/address this, could you reassign to someone else?

**Comment by Ashwin-Nikam** [comment]
@paulgildea  any updates on this? 😀

**Comment by paulgildea** [comment]
@mnajdova Is possible to get a repo of this issue?

**Comment by Ashwin-Nikam** [comment]
@paulgildea @mnajdova Do we have any updates on this?

**Comment by paulgildea** [comment]
@smhigley as FYI

**Comment by xugao** [comment]
@Ashwin-Nikam - if you simply press up/down arrow, the value is changing. In terms the key combo (control + option + shift + up/down arrow) VO told to use, it's indeed not working. 
@smhigley - could you advice how to address this?

**Comment by xugao** [comment]
@Ashwin-Nikam - I just notice this issue is happening in all the spin buttons and looks like this is a VO bug. You can repro the same issue in [this simple aria example](https://www.w3.org/TR/wai-aria-practices/examples/spinbutton/datepicker-spinbuttons.html)

**Comment by smhigley** [comment]
Just for more information: this isn't so much a VoiceOver bug as a gap in the language/platform. Until synthetic events for assistive tech are possible to create (i.e. through the AOM https://github.com/WICG/aom/blob/gh-pages/explainer.md#user-action-events-from-assistive-technology), custom spinbuttons will not be accessible to touch screen readers.

It just isn't possible to author a spinbutton (aside from the native `<input type="number">`) and handle the specific events used by mobile screen readers on the web today. That's why the increment/decrement buttons are still needed, and also why custom spinbuttons are generally discouraged 😃
